nanome/_internal/_shapes/_serialization/_mesh_serializer.py

```diff
@@ -1,5 +1,15 @@
 from nanome._internal._util._serializers import _TypeSerializer
 from nanome._internal._shapes._mesh import _Mesh
+from nanome.util import Logs
+
+import tempfile
+import os
+from io import BytesIO
+try:
+    from PIL import Image
```

Comment:
@ajm13 I was talking about that with Xavier.
So currently, nanome-lib has no dependency, it's not a hard requirement of course, but more a cool feature.
For images loading, PIL is required. This leaves us with two solutions:

1. The currently implemented one, adding PIL as an optional dependency and properly handling the case where PIL isn't installed. That's a valid solution, but kinda weird in terms of API: the user needs to import open3d in his plugin to load a model, but not PIL because nanome-lib handles that for him. Imo, that's an inconsistent API.
2. Expose a byte array for the user to fill properly with the texture he would have read by whatever mean he finds suitable. I'd favor that in terms of flexibility and API consistency.

What are your thoughts?

Reply:
Preferred 3rd solution:
- require texture_path to be `.png` (error if not `.png`)
- user can convert to `.png` on their own